newrelic/newrelic.go

```diff
@@ -55,15 +56,31 @@ func New(apiKey string, opts ...ConfigOption) (*NewRelic, error) {
 // ConfigOption configures the Config when provided to NewApplication.
 type ConfigOption func(*config.Config) error
 
-// ConfigRegion sets the New Relic Region this client will use
+// ConfigAPIKey sets the New Relic Admin API key this client will use.
```

Comment:
We should add a link to the docs site here, as the comment will go into godocs and folks might want some context on the API key types: https://docs.newrelic.com/docs/apis/get-started/intro-apis/types-new-relic-api-keys